assert/cmp/compare_test.go

```diff
@@ -248,9 +248,9 @@ func TestError(t *testing.T) {
 	result := Error(nil, "the error message")()
 	assertFailure(t, result, "expected an error, got nil")
 
-	result = Error(errors.New("other"), "the error message")()
+	result = Error(errors.Wrap(errors.New("other"), "wrapped"), "the error message")()
 	assertFailureHasPrefix(t, result,
-		`expected error "the error message", got other`)
+		`expected error "the error message", got "wrapped: other"`)
```

Comment:
Maybe this should include a trailing newline now as well, since that is being added by `formatErrorMessage()`. Otherwise we aren't really testing that the second branch is being hit properly.

Reply:
Fixed! I also added the first line of the dumped stack.